### PR TITLE
DOCS-3099 iOS Advanced Configuration Edits

### PR DESCRIPTION
**Pull request by alai97**
### What and why?

Adds a link to the source code for the `DDRUMMonitor` class + addresses copy nits.

### How?

DOCS-3099

### Review checklist

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration)
- [x] Make sure each commit and the PR mention the Issue number or JIRA reference

### CODE REVIEW

File: docs/rum_collection/advanced_configuration.md [hunk 1]
```diff
@@ -606,7 +611,7 @@ Each mapper is a Swift closure with a signature of `(T) -> T?`, where `T` is a c
 {{% /tab %}}
 {{< /tabs >}}
 
-Returning `nil` from the error, resource, or action mapper drops the event entirely (it won't be sent to Datadog). The value returned from the view event mapper must be not `nil` (to drop views customize your implementation of `UIKitRUMViewsPredicate`. Read more in [tracking views automatically](#automatically-track-views)).
+Returning `nil` from the error, resource, or action mapper drops the event entirely; the event is not sent to Datadog. The value returned from the view event mapper must be not `nil` (to drop views customize your implementation of `UIKitRUMViewsPredicate`. Read more in [tracking views automatically](#automatically-track-views)).
```
Comment (maycmlee) [inline docs/rum_collection/advanced_configuration.md:614]:
```suggestion
Returning `nil` from the error, resource, or action mapper drops the event entirely; the event is not sent to Datadog. The value returned from the view event mapper must not be `nil` (to drop views, customize your implementation of `UIKitRUMViewsPredicate`. Read more in [tracking views automatically](#automatically-track-views)).
```

**Review by maycmlee — APPROVED** [review]
A couple of small edits.

File: docs/rum_collection/advanced_configuration.md [hunk 2]
```diff
@@ -606,7 +611,7 @@ Each mapper is a Swift closure with a signature of `(T) -> T?`, where `T` is a c
 {{% /tab %}}
 {{< /tabs >}}
 
-Returning `nil` from the error, resource, or action mapper drops the event entirely (it won't be sent to Datadog). The value returned from the view event mapper must be not `nil` (to drop views customize your implementation of `UIKitRUMViewsPredicate`. Read more in [tracking views automatically](#automatically-track-views)).
+Returning `nil` from the error, resource, or action mapper drops the event entirely; the event is not sent to Datadog. The value returned from the view event mapper must not be `nil` (to drop views, customize your implementation of `UIKitRUMViewsPredicate`. Read more in [tracking views automatically](#automatically-track-views)).
```
Comment (ncreated) [inline docs/rum_collection/advanced_configuration.md:614]: We finish one sentence inside brackets and start a new one, which looks odd, no? How about:
```suggestion
Returning `nil` from the error, resource, or action mapper drops the event entirely; the event is not sent to Datadog. The value returned from the view event mapper must not be `nil` (to drop views, customize your implementation of `UIKitRUMViewsPredicate` - read more in [tracking views automatically](#automatically-track-views)).
```
or
```suggestion
Returning `nil` from the error, resource, or action mapper drops the event entirely; the event is not sent to Datadog. The value returned from the view event mapper must not be `nil` (to drop views, customize your implementation of `UIKitRUMViewsPredicate`; read more in [tracking views automatically](#automatically-track-views)).
```

**Review by ncreated — APPROVED** [review]
LG and good to go 💪, I left one suggestion.

**Pull request merged**